python/template.yml

```diff
@@ -938,6 +938,46 @@ sections:
             docstring: |
               Formulation of the Permittivity function. Currently only the
               exponential is used.
+  - name: GeometryOptimizer
```

Comment:
Is this the correct place for a detailed description of the sqnm parameters? If so, I can write a couple of sentences for each keyword.

Reply:
Yes, please write a few words on each of them here

Reply:
Ok, I will do that on my repository after I accepted the pull request